playground/frontend/pubspec.yaml

```diff
@@ -52,71 +32,30 @@ dependencies:
       ref: 9e2c9fe52a69481f038f4b6609e8a0a776429437
   collection: ^1.15.0
   expansion_widget: ^0.0.2
-  flutter:
-    sdk: flutter
-  flutter_localizations:
-    sdk: flutter
+  flutter: { sdk: flutter }
+  flutter_localizations: { sdk: flutter }
```

Comment:
Unified one-liners in this file make it easier to look for a package and to check if everything is sorted.